storagecharts2/src/main/php/templates/tpl.charts.php

```diff
@@ -42,11 +42,6 @@
 }
 ?>
 
-<div id="storagecharts2" data-sc-sort="<?php  print($scSort)?>" data-sc-size="<?php  print($huSize)?>" data-sc-size-hus="<?php  print($huSizeHus)?>">
```

Comment:
i understand the icon its sefl-explanatory.. but this line cares me.. i test just now and feedback in few minutes... if are good the sorting click div's then i'll commit this pull request.